test/index.spec.ts

```diff
@@ -27,6 +18,6 @@ describe('GraphQL API Tests', () => {
       `,
     });
 
-    equal(response.data.data.hello, 'Hello, world!');
+    expect(response.data.data.hello).to.equal('Hello, world!');
```

Comment:
mais uma questão de nomenclatura padrão dos projetos, geralmente usamos `expect to be equal`
é só questão de leitura, não tem impacto na lógica